### PR TITLE
Windows Docker NBS6 

### PR DESCRIPTION
**Pull request by njaved**

### CODE REVIEW

File: .github/workflows/Build-and-deploy-nbs6-win-container.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+on:
+  push:
+    branches:
+      - docker-nbs6-win
+    # paths:
+    #   - "cdc-sandbox/nbs6-win-container/**"
+jobs:
+  push-call-build-windows-container-workflow:
+    name: On Push Build Windows Container
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Build-windows-container.yaml@docker-nbs6-win
```
Comment (szamfir) [inline .github/workflows/Build-and-deploy-nbs6-win-container.yaml:11]: Need to change the reference from a branch to target version (or main)

**Review by szamfir — CHANGES_REQUESTED** [review]
See comment

File: .github/workflows/Build-and-deploy-nbs6-win-container.yaml (hunk 1 above)
Comment (njaved) [reply .github/workflows/Build-and-deploy-nbs6-win-container.yaml:11]: Updated to main

**Review by szamfir — APPROVED** [review]
LGTM

**Review by adamloup-enquizit — APPROVED** [review]
🪟 🐳

**Pull request merged**